packages/devtools_app/lib/src/code_size/code_size_screen.dart

```diff
@@ -96,4 +83,26 @@ class CodeSizeBodyState extends State<CodeSizeBody> with AutoDisposeMixin {
       return const SizedBox();
     }
   }
+
+  CodeSizeTable _buildTreeTable() {
```

Comment:
It is fine to just return `Widget` (probably more conventional) - here and below

Reply:
Done